### PR TITLE
Build .apk from examples

### PR DESCRIPTION
**Pull request by artemnikitin**

### CODE REVIEW

**Comment by artemnikitin** [comment]
Problem is that package name is part of credentials. Using different package name for every example will mean that we need to use different credentials for every example...

File: .ci/buildAll.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ ORIG_PWD="$PWD"
 
 for APP_PATH in $APP_PROJECTS; do
     cd $APP_PATH
-    ./gradlew build
+    ./gradlew assembleDebug
```
Comment (sschuberth) [inline .ci/buildAll.sh]: Why? `build` builds all build types (and thus depends on `assembleDebug`), plus it does run checks, if any, which is exactly what we should do in CI, IMO.

File: .ci/setCredentials.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+for file in $(find . -name AndroidManifest.xml -type f); do 
+	echo "Set credentials to:" $file
+	sed -i -- 's|{YOUR_APP_ID}|'"$SDK_EXAMPLES_APP_ID"'|g' $file
+	sed -i -- 's|{YOUR_APP_CODE}|'"$SDK_EXAMPLES_APP_TOKEN"'|g' $file
+	sed -i -- 's|{YOUR_LICENSE_KEY}|'"$SDK_EXAMPLES_LICENSE"'|g' $file
+done
+
+echo "Done"
+echo "-----------"
```
Comment (sschuberth) [inline .ci/setCredentials.sh]: Missing final new line.

File: .ci/collectApk.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ORIG_PWD="$PWD"
+
+mkdir $HOME/apk
+
+for APP_PATH in $APP_PROJECTS; do
+    cp $APP_PATH/app/build/outputs/apk/app-debug.apk $HOME/apk/${APP_PATH##*/}.apk
+done
+
+echo "Done"
+echo "-----------"
```
Comment (sschuberth) [inline .ci/collectApk.sh]: Missing final new line.

File: .ci/collectApk.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo "Collect apps..."
```
Comment (sschuberth) [inline .ci/collectApk.sh]: I'd rather set `addons.artifacts.paths` in `.travis.yml` instead of having this script.

**Comment by sschuberth** [comment]
> Using different package name for every example will mean that we need to use different credentials for every example...

Yes, exactly, but I don't see a (technical) problem with that.

**Comment by AmbroseLi** [comment]
Requiring different credentials shouldn't be an issue as the developer picking up sample code should be generating his/her own AppID/Code/LK from the dev portal anyhow.  The convention is that AppID/Code/LK isn't supplied with sample code.

**Comment by artemnikitin** [comment]
Not actual anymore...

**Pull request closed without merging**